### ISSUE TITLE
Request to Join ModernApps Ninja Github Org

### ISSUE THREAD

**Issue by rhr79** [ModernAppsNinja/modernappsninja.github.io#137]
#### By submitting this ticket, you acknowledge you have read and understood the registration instructions, and acknowledge and agree to the handling of the registration data you provide below as described in the registration instructions at the link above.
---

preferredEmailAddress: "richardrenardson@gmail.com”  
firstName: "Richard”  
lastName: "Renardson"  
twitterUsername: ""  
linkedinUsername: "rren79"  
isVmwareEmployee: true
vmwareEmailAddress: "renardsonr@vmware.com"

**Comment by afewell** [comment]
Hi @rhr79 apologies, normally member registration is a fully automated process, but for some reason your ticket didnt trigger the automation successfully, I am not sure why ... it definitely looks like you did everything right, let me look into what happened and I will figure it out and get your account fixed up, apologies for the inconvenience. Thanks!

**Comment by afewell** [comment]
Hi @rhr79 it looks like you ran into a strange error that rarely but sometimes happens with text files. If you look at the attached image, it appears that the quotation mark before your email and the quotation mark after your email address, they are both double quotes, but the closing quote is clearly different in appearance from the opening quote. I cant explain why this would happen, as while things like that are not uncommon on local apps when using different OS's, normally when you use the github web based editor I assumed it should always use consistent ascii codes and metacharacters, but apparently that is not always true. 

I will keep an eye out for root cause, but I think a simple fix could be if you wouldnt mind, if you could try opening another issue ticket with a request to join the org, I think that should work. I assume the reason why the closing quotation mark around your email was different is, perhaps you may have deleted one of the quotation marks that was already present in the form, so if you enter your email address carefully in between the existing quotation marks without deleting or replacing the quotations marks, then I think that should work. I apologize for the inconvenience, I have never seen github have this issue before and thus far your the first person to have this problem, but if it happens more I will change the process to ensure its reliable for everyone. 

So in summary, if you could please open another issue ticket and carefully fill out the fields trying not to delete or replace the quotation marks that are already in the issue ticket, then I think that should work, but if it does not please open an issue to make sure I know, and I will be sure to get you fixed up one way or another. 

Sorry for the inconvenience, and thanks so much for your interest in the program!

**Comment by afewell** [comment]
@rhr79 I looked into this a bit more, and it appears that there are 3 different types of double quotes possible in ascii code:
```
"   U+0022 QUOTATION MARK
“   U+201C LEFT DOUBLE QUOTATION MARK
”   U+201D RIGHT DOUBLE QUOTATION MARK
```
it appears that the closing quote for the email address and for the first name are the right double quotation mark, whereas all the rest of the double quotes in this document are just a standard quotation mark. It is likely something in your OS or system settings that have your keyboard enter the right double quote mark instead of the standard quotation mark. I am not sure exactly how to ensure that your keyboard could be made to enter the correct type of quotation mark, but I think my suggested resolution above should work ... as you can see in the image above, only 2 of the quotation marks in the form were the wrong type and so it would seem you may have deleted and replaced those quotation marks while you filled the form, so I think if you enter the form again being careful not to delete or replace any of the quotation marks. If that method did not work, other options could be trying to find how to ensure your keyboard enters the correct type of quotation ascii, or alternatively, you could try replacing ALL of the quotation marks, as I think the workflow should work with any type of quotation mark but may be having a problem because there are multiple different types of double quote characters in the document. 

Please let me know once you have had the chance to enter another registration ticket, I will keep a close eye on this to help ensure you get registered. Sorry again for the inconvenience!

**Comment by rhr79** [comment]
Thanks so much for the time @afewell looking at this - I filled it out on my phone and deleted the quote indeed so should have caught that, apologies. I've resubmitted and the invitation has arrived this time so I'm probably good. Thanks again!

**Comment by afewell** [comment]
@rhr79 awesome looks like your member site is up so I assume everything worked out, if you run into any issues or have any questions let me know, issue tickets are great or vmware slack works great to. 

Thanks!

**Comment by afewell** [comment]
one thing I need to document better is when you register/enroll in a course or submit a test, you submit the document per the instructions, and then you just wait 1-2 minutes while the process runs, I dont think I made the waiting part clear in the instructions but I made some videos I will post tomorrow that walk through the processes and hopefully help improve the docs. 

Thanks!
